azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/HandlerFactoryImpl.java

```diff
@@ -94,6 +125,12 @@ public ArtifactHandler getArtifactHandler(final AbstractWebAppMojo mojo) throws
         }
     }
 
+    protected ArtifactHandler getArtifactHandlerFromV2Schema (final AbstractWebAppMojo mojo)
+        throws MojoExecutionException {
+        // todo
+        throw new MojoExecutionException("Unimplemented for schema version: " + mojo.getSchemaVersion());

```

Comment:
Unavailable schema version: ....

Reply:
Will replace the todo in next PR of implementation, do you think we still need to update for the words?

Reply:
@SummerSun Please also involve @Matthew-Dong to discuss about this change. Since we may also want to release the new plugin to support zip deploy ASAP. So it is worth discussing whether we need to include this change into the release.

Reply:
I think we all agree that this will not be included in the zip deploy.